### PR TITLE
[TEST] increase coverage of project preference service test

### PR DESCRIPTION
**Pull request by abzymeinsjtu**
## Purpose of the pull request

increase coverage of project preference service test

### CODE REVIEW

**Review by EricGao888 — DISMISSED** [review]

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/ProjectPreferenceServiceTest.java [hunk 1]
```diff
@@ -60,28 +60,65 @@ public class ProjectPreferenceServiceTest {
     public void testUpdateProjectPreference() {
         User loginUser = getGeneralUser();
 
+        // NO PERMISSION
```
Comment (Radeity) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/ProjectPreferenceServiceTest.java]: Please unify the case in comments :D

**Review by Radeity — APPROVED** [review]
LGTM

**Review by EricGao888 — APPROVED** [review]

**Comment by EricGao888** [comment]
> LGTM

@Radeity Could u plz help add a ready-to-merge label to this PR? I'm using github ios app and could not add labels without accessing my laptop. Thx

**Comment by Radeity** [comment]
> @Radeity Could u plz help add a ready-to-merge label to this PR? I'm using github ios app and could not add labels without accessing my laptop. Thx

Done.

**Pull request merged**